src/Pdp/PublicSuffixListManager.php

```diff
@@ -63,10 +70,22 @@ public function __construct($cacheDir = null)
     public function refreshPublicSuffixList()
```

Comment:
I'll will refactor this method but this will include deprecating `PublicListSufficMananger::parseListToArray` as this method is public and only return the full PDP list